_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+
+# Support your curators
+
+Just because a twitter account has >20K followers, the likes, re-tweets, and comments don't come automatically. You still have to earn engagement with your content. Many of the tweets I sent had little engagement, and that is okay. Supporting your curators by engaging with their tweets or sending notes of encouragement is _much_ appreciated. I thank **everyone** who engaged with me during my curation, with a special shout out to Alison Hill who re-energized me mid-week with comments on the R-Ladies bloggers [thread](https://twitter.com/WeAreRLadies/status/1362021673239785473){target="_blank"}. 
+
+I cannot emphasize this enough: every like, re-tweet, comment, and direct message helps. In addition, if you have curated in the past, consider sending new curators a personal welcome message and an invitation to ask you any questions.
+
+
+# Reflection
+
+Between prepared and ad-hoc content and discussions with followers, I tweeted a lot! (At least for me.)  Here is a [summary thread](https://twitter.com/spcanelon/status/1363518469782843396){target="_blank"} of my tweets for the week. My tweets were not perfect, and that is okay. I messed up threads, had typos, and shared deprecated code, among other things. Check out my [blooper reel](https://twitter.com/PipingHotData/status/1364183660744896513){target="_blank"} for tweets that I bungled.
+
+Serving as a curator was intimidating and time consuming, but I am very glad I did it. Many good things have happened as direct result of that week, including: 
+
+* [discussing](https://twitter.com/WeAreRLadies/status/1363144545677017089){target="_blank"} comparisons between `{blogdown}` and `{distill}` with Alison Hill 
+
+* [collaborating](https://www.pipinghotdata.com/posts/2021-03-08-r-ladies-styled-code-gifs-with-xaringan-and-flipbookr/){target="_blank"} with Silvia Canelón to style code gifs
+
+* engaging with new people on twitter that I want to continue to engage with
+
+* learning about valuable new-to-me packages, functions, and work flows 
+
+* being invited to [speak for R-Ladies Miami](https://github.com/shannonpileggi/growing-into-r){target="_blank"}
+
+* seeing my "Asking for help online" content re-used in Sharla Gelfand's ["make a reprex... please"](https://twitter.com/sharlagelfand/status/1365665149063987201){target="_blank"} presentation
+
+* co-developing a unit testing workshop with [Gordon Shotwell](https://twitter.com/gshotwell/status/1362769538631221251){target="_blank"}
+
+During my curating week I tried to embody the tweets that I value: honest questions, thoughtful discussion, generous sharing, supportive community, and humorous exchanges. And to borrow from Vicki Boykis in the rstudio::global(2021) [keynote](https://rstudio.com/resources/rstudioglobal-2021/your-public-garden/){target="_blank"}, I created my own public garden that cultivated new connections and collaborations. 
+
```

Comment:
A last question I'd have is, did this experience influence your tweeting from your personal account?

Reply:
yessss good question!